ui/src/main/resources/WEB-INF/web.xml

```diff
@@ -97,6 +97,27 @@
     </error-page>
 
     <filter>
+        <filter-name>DoSFilter</filter-name>
```

Comment:
Do we still need this? We definitely should not be checking in the jars.

Reply:
That was in the usage of the filter but it will also work if I add it as an embedded dependency. We still need this filter here though or else it won't filter the GET requests.